sass/components/_collection.scss

```diff
@@ -87,20 +89,24 @@
       border-bottom: 1px solid $collection-border-color;
       padding: 10px 20px;
     }
+
     .collection-item {
       padding-left: 30px;
     }
+
     .collection-item.avatar {
       padding-left: 72px;
     }
   }
 
 }
+
 // Made less specific to allow easier overriding
 .secondary-content {
   float: right;
-  color: $secondary-color;
+  color: $primary-color;
 }
```

Comment:
Whats here the usage of primary?

Reply:
I saw that you changed that other places too.
I assume you first had primary and then adapted for a better appearance.
Or was this due to the material specs?

Reply:
Yes, I tried to align everything to materialize specs, where they use primary color almost everywhere, and secondary color is used only as accent for important actions (like floating action button)